sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SessionRecords/SchemaRegistryClientLiveTests/CanGetSchema(%Json%).json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Did we need to re-record these because of the parameter changing the name? 
Are there extra recordings that need to be deleted?

Reply:
Since the API changed that means that all of them need to be re-recorded?